src/align/anchors.jl

```diff
@@ -0,0 +1,271 @@
+
+# Anchor definition
+# ------------------
+"""
+A type to store the operation enocded in an alignment (CIGAR Operations).
+Also stores the position in the alignment view of the sequences, and the
+corresponding position in the unaltered source sequence (nucleotide or protein).
+
+It stores the position values as Int types and the alignment operation is stored
+as a type `Operation`, these are defined in the file `operations.jl` and loaded
+into Bio.Align namespace as a series of global constants.
+
+Calling the AlignmentAnchor default constructor with default arguments sets both
+integer fields to 0, and the operation field to the global constant OP_INVALID.
+"""
+immutable AlignmentAnchor
+    seqpos::Int
+    refpos::Int
+    op::Operation
+end
+
+
+# Basic operators for AlignmentAnchors
+# -------------------------------------
+
+"""
+Print to screen or other IO stream, a formatted description of an
+AlignmentAnchor.
+"""
+function show(io::IO, anc::AlignmentAnchor)
```

Comment:
I am thinking of the documentation generated for the site - does Judo fill in the base documentation?